Testscripts/Windows/RemoveHardDisk.ps1

```diff
@@ -42,11 +42,17 @@ function Main {
         }
 
         $field_value = $fields[0].Trim().ToLower()
+        if ($field_value -match "scsi_") {
```

Comment:
'-match' or '-like'?

Reply:
@juhlee-microsoft , both ok, if use -match, it is "if ($field_value -match "scsi_")", if use -like, need to use if ($field_value -like "scsi_*")
In this file, no other place use $Matches value, so both ok.